CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## v2.39.0 - 14 Jun 2024
 
 ### Enhancements
+
+* [vrops] Issue 280 / Support for priority ordering of the vROPs policies during push.
```

Comment:
2.39.0 is already in release. Please move this above the release number so it is added as part of the next release